tools/mapping/gen_mapping_files/runoff_to_ocn/runoff_map.nml

```diff
@@ -0,0 +1,15 @@
+  &input_nml
+   gridtype     = 'rtm'
+   file_roff    = '/glade/p/cesm/cseg/inputdata/lnd/clm2/rtmdata/rdirc.05.061026'
+   file_ocn     = '/glade/p/cesm/cseg/mapping/grids/gx3v7_120309.nc'
+   file_nn      = 'map_r05_to_gx3v7_nn_130507.nc '
+   file_smooth  = 'map_gx3v7_to_gx3v7_sm_e1000r300_130507.nc '
```

Comment:
What's the source of these tools/mapping changes? I don't see any recent changes on the ACME side,